docs/source/common/optimizers.rst

```diff
@@ -32,25 +44,40 @@ to manually manage the optimization process. To do so, do the following:
 
         def __init__(self):
             super().__init__()
-            # Important: This property activate ``manual optimization`` for your model
+            # Important: This property activates manual optimization.
             self.automatic_optimization = False
 
         def training_step(batch, batch_idx):
             opt = self.optimizers()
+            opt.zero_grad()
             loss = self.compute_loss(batch)
             self.manual_backward(loss)
+            opt.step()
 
-.. note:: This is only recommended for experts who need ultimate flexibility. Lightning will handle only precision and accelerators logic. The users are left with ``optimizer.zero_grad()``, gradient accumulation, model toggling, etc..
+.. warning::
+   Before 1.2, ``optimizer.step()`` was calling ``optimizer.zero_grad()`` internally.
+   From 1.2, it is left to the user's expertise.
 
-.. warning:: Before 1.2, ``optimzer.step`` was calling ``optimizer.zero_grad()`` internally. From 1.2, it is left to the users expertise.
+.. tip::
+   * ``self.optimizers()`` will return :class:`~pytorch_lightning.core.optimizer.LightningOptimizer` objects. You can
+     access your own optimizer with ``optimizer.optimizer``. However, if you use your own optimizer to perform a step,
+     Lightning won't be able to support accelerators and precision for you.
+   * Be careful where you call ``optimizer.zero_grad()``, or your model won't converge.
+     It is good practice to call ``optimizer.zero_grad()`` before ``self.manual_backward(loss)``.
```

Comment:
Remove this now that we have it at the bottom?

```suggestion
   Be careful where you call ``optimizer.zero_grad()``, or your model won't converge.
   It is good practice to call ``optimizer.zero_grad()`` before ``self.manual_backward(loss)``.
```

Reply:
I would leave it in both manual and automatic optimization sections as one might only look at a section of their interest, but the decision is yours. Should I apply your suggestion here?